org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/ScanningPerspective.java

```diff
@@ -53,7 +53,7 @@ public void createInitialLayout(IPageLayout layout) {
 		{
 			IFolderLayout folderLayout = layout.createFolder("folder_0", IPageLayout.TOP, 0.66f, "org.eclipse.scanning.device.ui.spectrumview");
 			folderLayout.addView("org.eclipse.scanning.device.ui.vis.visualiseView");
-			folderLayout.addView("org.eclipse.scanning.device.ui.vis.StreamView");
+			//folderLayout.addView("org.eclipse.scanning.device.ui.vis.StreamView");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)